app/controllers/stripe_active_cards_controller.rb

```diff
@@ -11,16 +11,14 @@ def create
       Rails.logger.info("Stripe Add New Card Success - #{current_user.username}")
       flash[:settings_notice] = "Your billing information has been updated"
     else
-      DatadogStatsClient.increment("stripe.errors")
+      DatadogStatsClient.increment("stripe.errors", tags: ["event:new_subscription", "user_id:#{current_user.id}"])
```

Comment:
the name "new_subscription" is not exactly correct :D this is just a "new card" or a "new source". Stripe has the concept of subscriptions but it's another thing and it's not handled by this endpoint.

See:

- https://stripe.com/docs/api/sources (which is what we're creating or updating in this controller)
- https://stripe.com/docs/api/subscriptions (which is a feature we used to have but don't anymore, as there's no concept of recurring membership on DEV right now), see https://github.com/thepracticaldev/dev.to/pull/2741

cc @mstruve

Reply:
Thanks for the context @rhymes! probably a good idea to update the tag then